internal/app/api/v1/scripts.go

```diff
@@ -173,21 +175,34 @@ func (s kubtestAPI) ExecuteScript() fiber.Handler {
 			execution,
 			request.Params,
 		)
-		s.Repository.Insert(ctx, scriptExecution)
+		err = s.Repository.Insert(ctx, scriptExecution)
+		if err != nil {
+			return s.Error(c, http.StatusBadGateway, fmt.Errorf("inserting script CR error: %w", err))
 
+		}
+		fmt.Println("*******ID:", scriptExecution.Id)
+		jobClient, err := jobs.NewJobClient()
+		if err != nil {
+			return s.Error(c, http.StatusInternalServerError, fmt.Errorf("can't get k8s client: %w", err))
+		}
 		// save watch result asynchronously
-		go func(scriptExecution kubtest.ScriptExecution, executor client.HTTPExecutorClient) {
+		go func(scriptExecution kubtest.ScriptExecution) error {
```

Comment:
This will not work as HTTP client, There is Watch method which handles status change

Reply:
yes i'm aware of that but overall process was quite different so we will have to make this better